Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use Alpine-node as a base image because it is much smaller.
+#
+# Base image size for mhart/alpine-node:4 is 13 MB
+# Base image for standard node:4 is 253 MB.
+#
+# Our production deploy does not have any C/C++ library dependencies yet
+# so this is still easy to do.
+FROM mhart/alpine-node:4
```

Comment:
Most of the image makes sense to me, but I really can't figure out what this command does:
https://github.com/mhart/alpine-node/blob/01ed9039d53628391fe4ccae4ae9dfd32797d701/Dockerfile#L30
Looks like it makes sure that the executable can't do something to memory, but not following 100%. It's probably fine, just curious why this image was chosen.